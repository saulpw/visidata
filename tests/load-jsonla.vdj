#!vd -p
{"longname": "open-file", "input": "sample_data/officials.jsonla", "keystrokes": "o"}
{"sheet": "officials", "col": "", "row": "", "longname": "columns-sheet", "input": "", "keystrokes": "Shift+C", "comment": "open Columns Sheet: edit column properties for current sheet"}
