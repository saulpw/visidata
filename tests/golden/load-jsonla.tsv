name	width	height	type	fmtstr	formatter	value	expr	aggregators
Name		1				Daniel	0	
Number		1	int			1	1	
Exile		1	bool			True	2	
